### PR TITLE
Add flare and songbird

### PR DESCRIPTION
**Pull request by jO-Osko**

### CODE REVIEW

File: docs/sdk/helper-sdks/providers/evm.mdx [hunk 1]
```diff
@@ -112,6 +112,8 @@ Some of the commonly used L2s and the Ethereum chain ids are listed below.
 | 0x89   | 137     | Polygon Mainnet             |
 | 0xA86A | 43114   | Avalanche C-Chain           |
 | 0xA    | 10      | Optimism                    |
+| 0xE    | 14      | Flare                       |
+| 0x13   | 19      | Flare                       |
```
Comment (ihsraham) [inline docs/sdk/helper-sdks/providers/evm.mdx:116]: Wouldn't the network be Songbird for 0x13? @jO-Osko

Comment (jO-Osko) [reply docs/sdk/helper-sdks/providers/evm.mdx]: It would, fixed.

**Review by ihsraham — APPROVED** [review]
LGTM

**Comment by jO-Osko** [comment]
I still need to add banner images.

File: docs/connect-blockchain/multi-chain.mdx [hunk 2]
```diff
@@ -114,6 +114,8 @@ the Ethereum provider.
     { label: "Binance Smart Chain", value: "bsc" },
     {label: "Avalanche", value: "avalanche"},
     {label: "Cronos", value: "cronos"},
+    {label: "Flare", value: "flare"},
```
Comment (yashovardhan) [inline docs/connect-blockchain/multi-chain.mdx:117]: Unfortunately we cannot add this in the multi-chain docs. That is particular to an example that we have and we do not want to change it.

**Review by yashovardhan — COMMENTED** [review]
I'll be adding the banner images, alongside deleting the changes in the multi chain doc.

**Pull request merged**